### PR TITLE
support newer Ubuntu versions

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -28,7 +28,8 @@
           :alias  => 'tftp-server',
         })
 
-        if facts[:operatingsystem] == 'Debian' && facts[:operatingsystemrelease].start_with?('8.')
+        if (facts[:operatingsystem] == 'Debian' && facts[:operatingsystemrelease].start_with?('8.')) ||
+           (facts[:operatingsystem] == 'Debian' && facts[:operatingsystemrelease].start_with?('16.'))
```
Comment (ekohl) [inline spec/classes/init_spec.rb]: Shouldn't this be Ubuntu?

Comment (mmoll) [inline spec/classes/init_spec.rb]: indeed.

**Comment by ekohl** [comment]
:+1:

**Comment by GregSutcliffe** [comment]
Merged as 176e0ba, ty @mmoll

**Pull request closed without merging**